app/views/devise/shared/_flashes.html.erb

```diff
@@ -1,11 +1,17 @@
-<% if notice %>
-  <div>
-    <%= notice %>
-  </div>
-<% end %>
+<div class="devise-alert" data-controller="hide-devise-alert" data-hide-devise-alert-target="alert">
```

Comment:
If you use this.element in the controller you can remove the data-target here

Reply:
will be done on the next commit